test/simplifyExpression/breakUpNumeratorSearch/breakUpNumeratorSearch.test.js

```diff
@@ -11,6 +11,13 @@ describe('breakUpNumerator', function() {
     ['(x+3+y)/3', '(x / 3 + 3/3 + y / 3)'],
     ['(2+x)/4', '(2/4 + x / 4)'],
     ['2(x+3)/3', '2 * (x / 3 + 3/3)'],
+    ['(2x + 3)/(2x + 2)', '((2x + 2) / (2x + 2) + 1 / (2x + 2))'],
+    ['(2x - 3)/(2x + 2)', '((2x + 2) / (2x + 2) - 5 / (2x + 2))'],
+    ['(2x + 3)/(2x)', '(2x / (2x) + 3 / (2x))'],
+    ['(3 + 2x)/(2x)', '(3 / (2x) + 2x / (2x))'],
+    ['(4x + 3)/(2x + 2)', '(2 * (2x + 2) / (2x + 2) - 1 / (2x + 2))'],
+//    ['(2x)/(3 + 2x)', '((3 + 2x) / (3 + 2x) - 3 / (3 + 2x))'],
+//    ['(2x)/(2x + 3)', '((2x + 3) / (2x + 3)) - 3 / (2x + 3)'],
```

Comment:
if these tests pass, uncomment them

otherwise leave a TODO with what we'll need to do in the future to make them pass